doc/reference/autofiles/top_level/blosc2.Tuner.rst

```diff
@@ -0,0 +1,13 @@
+﻿blosc2.Tuner
+============
+
+.. currentmodule:: blosc2
+
+.. autoclass:: Tuner
+   
+   .. rubric:: Attributes
+
+   .. autosummary::
+   
+      ~Tuner.STUNE
+      ~Tuner.BTUNE
```

Comment:
I'd add some docstrings here:

STUNE: A 'simple' tuner.  This is the default in the Blosc2 library

BTUNE: A more sophisticated tuner that can select different codecs/filters for different chunks.  More info: https://btune.blosc.org

Reply:
Fixed in 5d6795a2f591109e1b70a1bca2cbeb6c11844cb5